src/pages/Account/Tabs/ModulesTab.tsx

```diff
@@ -126,14 +126,7 @@ function ModulesContentReworked({address}: {address: string}): JSX.Element {
 
```

Comment:
this change lgtm. but the mobile view's broken, can we quickly fix it as well?

<img width="299" alt="Screenshot 2023-03-03 at 9 58 27 AM" src="https://user-images.githubusercontent.com/121921928/222793625-fe2bec97-70a8-4003-8d84-77b95a59da89.png">

Reply:
what's the expected look?